src/main/java/database/Storage.java

```diff
@@ -0,0 +1,12 @@
+package database;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class Storage {
+    private static Map<String, Integer> assortment = new HashMap<>();
+
+    public static Map<String, Integer> getAssortment() {
+        return assortment;
+    }
```

Comment:
Returning a reference to a mutable object stored in one of the object's fields exposes the internal representation of the object. It's a violation of encapsulation. To fix this, return a copy of the object instead.